src/core/Hocon/Akka.Hocon.csproj

```diff
@@ -1,10 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.4;net461</TargetFrameworks>
+    <TargetFrameworks>net45;netstandard1.4</TargetFrameworks>
```

Comment:
because that's what we're targeting in v1.3 branch (current) in the main akka.net repo